WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3986,7 +4025,7 @@ public void onEventMainThread(UploadService.UploadMediaRetryEvent event) {
 
     private boolean shouldShowAsyncPromoDialog() {
```

Comment:
Note: this function will be re-done or used somewhere else (secondary option) if we go this route.

Reply:
Done in 51dccbb - note the "Schedule with confidence" promo dialog was never showing before.